app/src/app/components/specmath-modal/defaults-file-upload/defaults-file-upload.component.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+import { DefaultsFileUploadOptions } from '../../../../shared/interfaces';
+
+@Component({
+  selector: 'app-defaults-file-upload',
+  templateUrl: './defaults-file-upload.component.html',
+  styleUrls: ['./defaults-file-upload.component.scss']
+})
+export class DefaultsFileUploadComponent implements OnInit  {
+  step2Options?: DefaultsFileUploadOptions = {
```

Comment:
Similar to what I've commented before. Is there something this could be named that describes the purpose of the variable (e.g. upload options) instead of the step it's used in?